Initialtask/src/Service/ProductService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Service;
+
+use App\Repository\ProductRepository;
+
+class ProductService
```

Comment:
probably you'll neede to extend ServiceEntityRepository
Please read this article https://newbedev.com/symfony-inject-doctrine-repository-in-service